test_gen_scripts/utils.py

```diff
@@ -41,11 +41,10 @@ def get_good_signature(message: bytes, hash=False):
     return (px, py, r, s)
 
 
-def get_raw_signature(message: bytes):
-    seed = generate_next_seed()
-    sk = generate_deterministic_key(seed)
+def get_raw_signature(message: bytes, sk):
```

Comment:
Why is there `sk` as a function argument and what does it do?
It is shadowed here `sk = generate_deterministic_key(generate_next_seed())`